Source/DECUtilRawByteStringHelper.pas

```diff
@@ -44,7 +44,7 @@ implementation
   StrRec = packed record
   {$IF defined(CPU64BITS)}
```

Comment:
Is it technically possible to change this to {$IFDEF CPU64BITS}?

Reply:
Sure.

Reply:
Thanks for confirmation. Done.